test/examples/boostrap.js

```diff
@@ -0,0 +1,184 @@
+/* global process, Promise, before, after */
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
It's only used here, because `puppeteer` must be manually installed (the other tes files don't directly depend on `puppeteer` so they don't need the `eslint-disable` trick).
So I'd say we can keep it here, WDYT?

Reply:
Ah OK; yeah, keep it that way then.